### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by antoinemeyer**
https://trizic.atlassian.net/browse/HOME-292

I have removed the pojo generation through gradle as it wasn't used for consumer anyway. Let me know if you want to restore it.

@abuecker

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,114 @@
-# api-docs-consumer
\ No newline at end of file
+# API Docs for consumer
+
+[![wercker status](https://app.wercker.com/status/71c5fdd2c379b09f18786c1cf741b243/m "wercker status")](https://app.wercker.com/project/bykey/71c5fdd2c379b09f18786c1cf741b243)
+
+Live: http://trizic-api-docs.s3-website-us-east-1.amazonaws.com
```
Comment (abuecker) [inline README.md]: Need the new `url`

File: package.json~ [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+  "name": "trizic-api-docs",
```
Comment (abuecker) [inline package.json~]: New package name.

File: README.md [hunk 3]
```diff
@@ -1 +1,114 @@
...
+
+    npm i http://trizic-npm.s3.amazonaws.com/api-docs-consumer/api-docs-consumer-<VERSION>.tgz --save
+
+## Wercker
+
+This uses [Wercker](http://wercker.com) for Continuous Deployment.  On
+successful builds of the master branch, the documentation will be automatically
+deployed to:
+
+http://trizic-api-docs.s3-website-us-east-1.amazonaws.com
```
Comment (abuecker) [inline README.md]: New `url`

**Comment by abuecker** [comment]
No need to put back in the POJO/ graddle.

File: api/index.raml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#%RAML 0.8
+title: Trizic
+baseUri: https://api.trizic.com
```
Comment (olmstead) [inline api/index.raml]: Base URI should match were we plan to run this from.

Comment (antoinemeyer) [inline api/index.raml]: I believe that would be same, we're just splitting the RAML spec, not touching the actual api.
@abuecker you agree?

**Comment by antoinemeyer** [comment]
@abuecker please review

**Comment by abuecker** [comment]
👍

**Pull request merged**